interface/attribute.yaml

```diff
@@ -0,0 +1,13 @@
+components:
+  schemas:
+    Attribute:
+      description: >-
+        attributes of the device interfaces that specify an internal architecture behind the interface, for example: NPU #, NPU Core, Slice #
+      type: object
+      properties:
+        name:

```

Comment:
To avoid confusion between `name` used in devices and interfaces, suggest to call this as `key`

Reply:
Updated